edx_django_utils/cache/utils.py

```diff
@@ -259,6 +260,10 @@ def _get_cached_response_from_django_cache(key):
             return CachedResponse(is_found=False, key=key, value=None)
 
         cached_value = django_cache.get(key, _CACHE_MISS)
+
+        if cached_value is None and not TieredCache._is_forced_cache_miss_for_none_disabled():
```

Comment:
Where did my `_CACHED_NONE` code go? It seemed like simple code to make handling of caching `None` consistent for any memcached library, so I'd like to see it restored. Additionally, when the cached value is `_CACHED_NONE` on retrieval, I wanted to see something like `set_custom_attribute('retrieved_cached_none, True)`, regardless of the value of the switch.

Reply:
`_CACHED_NONE` functionality has been added again. 

`set_custom_attribute('retrieved_cached_none, True)` was removed because it was giving `cyclic-import` related errors. I would add it in a `separate` commit.